src/aux/backend.rs

```diff
@@ -0,0 +1,114 @@
+/// Removable block devices
```

Comment:
I'm not sure why the source code is here as well?

Reply:
Please refer to https://github.com/flathub/flathub/pull/2002#issuecomment-748593489 and https://discourse.flathub.org/t/flatpak-builder-cannot-find-git-source-files/980

Primarily, I only had the manifest file here. But Cargo was unable to find any Cargo.toml file or even the generated dependency file even though they exist in the upstream repo. Hence I decided to hard-sync the build files to work around it.

Reply:
It failed because you forgot to add the generated json file here in-tree. The rest of files are not needed.

Reply:
Okay, I removed all the unnecessary sources and build files.

Reply:
It seems like you have pushed .flatpak-builder files

Reply:
Already removed them see https://github.com/flathub/flathub/pull/2002/commits/07b8cb48ce72743b5133b182d45c8de10e3ce075

Reply:
The resources directory is still here though